src/uiFactory/uiFactory.ts

```diff
@@ -0,0 +1,13 @@
+import type {UIFactory} from './types';
+
+const uiFactoryBase: UIFactory = {};
+
+export function configureUIFactory(overrides: UIFactory) {
```

Comment:
Approach inspired by ytsaurus-ui

https://github.com/ytsaurus/ytsaurus-ui/blob/main/packages/ui/src/ui/UIFactory/index.tsx#L476

I think it is easier and more straightforward to register functions here than to pass it from props.

Function for monitoring and logs links could be added here too